timemachine/fe/rbfe.py

```diff
@@ -343,9 +344,12 @@ def _optimize_coords_along_states(initial_states: List[InitialState]) -> List[ND
     for idx, initial_state in enumerate(initial_states):
         print(f"Optimizing initial state at λ={initial_state.lamb}")
         free_idxs = get_free_idxs(initial_state)
-        x_opt = optimize_coords_state(
-            initial_state.potentials, x_opt, initial_state.box0, free_idxs, assert_energy_decreased=idx == 0
-        )
+        try:
+            x_opt = optimize_coords_state(
+                initial_state.potentials, x_opt, initial_state.box0, free_idxs, assert_energy_decreased=idx == 0
+            )
+        except (AssertionError, minimizer.MinimizationError) as e:
```

Comment:
This was just a helpful additional message to tell me which lambda window was actually blowing up.